### ISSUE TITLE
bigrquery issue with Rcurl and httr

### ISSUE THREAD

**Issue by hidewawa** [rstats-db/bigrquery#75]
I am using bigrquery version 0.1.0.9000 version. I found two issues - 

1. I have Rcurl version 1.96 - 0, which is required for my other work. it seems like bigrquery doesn't work with 1.96, but 1.95 version is fine. Will this be fixed?

2. I maybe stored some bad credential in .httr in the first time, and when I used reset_access_cred, the old credential persisted. I get 'invalid credentials' error message. When I manually removed the .httr file, it works fine.

**Comment by hadley** [comment]
bigrquery doesn't use RCurl.

OAuth credential caching is a work in progress.

**Comment by hidewawa** [comment]
When I ran it in the Rstudio, it actually crashs, so I cannot see the error message. So I run it in R, here's the error - 
Error: 
In addition: Warning message:
In mapCurlOptNames(names(.els), asNames = TRUE) :
  Unrecognized CURL options: writedata

Even though Rcurl is not used in bigrquery. But seems like maybe some libcurl function is called. When I rolled back Rcurl to previous version. The problem seems go away.
